it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/StartControllerISpec.scala

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmrc.agentsubscriptionfrontend.controllers
+
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{contentType, _}
+import uk.gov.hmrc.agentsubscriptionfrontend.stubs.AuthStub
+import uk.gov.hmrc.agentsubscriptionfrontend.support.SampleUsers._
+
+class StartControllerISpec extends BaseControllerISpec {
+
+  private lazy val controller: StartController = app.injector.instanceOf[StartController]
+
+  "context root" should {
+    "redirect to start page" in {
+      val result = await(controller.root(FakeRequest()))
+
+      status(result) shouldBe 303
+      redirectLocation(result).head should include ("/start")
+    }
+  }
+
+  "start" should {
+    "not require authentication" in {
+      AuthStub.userIsNotAuthenticated()
+
+      val result = await(controller.start(FakeRequest()))
+
+      status(result) shouldBe 200
+    }
+
+    "be available at /agent-subscription/start" in {
```

Comment:
I'm going to have a quick look to see if we can pass session keys into `route()` - if it's easy it would be nice for our tests to be able to use it (e.g. to cover URLs)